src/FSharp.Core/prim-types.fs

```diff
@@ -374,6 +374,59 @@ namespace Microsoft.FSharp.Core
     type NoCompilerInliningAttribute() =
         inherit Attribute()
 
+#if !NET5_0_OR_GREATER
+namespace System.Diagnostics.CodeAnalysis
+
+    open System
+    open Microsoft.FSharp.Core
+
+    /// <summary>
+    /// Specifies the types of members that are dynamically accessed.
+    ///
+    /// This enumeration has a <see cref="FlagsAttribute"/> attribute that allows a
+    /// bitwise combination of its member values.
+    /// </summary>
+    [<Flags; RequireQualifiedAccessAttribute>]
+    type internal DynamicallyAccessedMemberTypes =
+        | None = 0
+        | PublicParameterlessConstructor = 0x0001
+        | PublicConstructors = 0x0003
+        | NonPublicConstructors = 0x0004
+        | PublicMethods = 0x0008
+        | NonPublicMethods = 0x0010
+        | PublicFields = 0x0020
+        | NonPublicFields = 0x0040
+        | PublicNestedTypes = 0x0080
+        | NonPublicNestedTypes = 0x0100
+        | PublicProperties = 0x0200
+        | NonPublicProperties = 0x0400
+        | PublicEvents = 0x0800
+        | NonPublicEvents = 0x1000
+        | All = 0xffffffff
```

Comment:
I can see that this PR only uses "All".
If by design, do we need to shim the full list in that case?

Reply:
That's a good question. I will leave them in, but commented out, so they are visible, if someone in the future decides they need some other subset.